src/models/usersDao.js

```diff
@@ -19,6 +29,30 @@ const signUp = async (email, password, points) => {
   }
 };
 
+const logIn = async (email) => {
+  try {
+    return await dataSource.query(
+      `SELECT
+      id,
+      email,
+      password,
+      points,
+      first_name,
+      last_name,
+      subscription
+      FROM users
+      WHERE users.email = ?`,
+      [email]
+    );
```

Comment:
1. user의 data를 주는 것이기 때문에 배열이 아닌 객체 형태로 return 될 수 있게 수정해 주세요
```suggestion
    const [user] = await dataSource.query(
      `SELECT
      id,
      email,
      password,
      points,
      first_name,
      last_name,
      subscription
      FROM users
      WHERE users.email = ?`,
      [email]
    );
    
    return user
```